### PR TITLE
Added forward url's and local url's

### PR DESCRIPTION
**Pull request by WlinkNET**
Just check pnpm-lock.yaml didnt change after rebase

### CODE REVIEW

File: app/components/block/Block.client.tsx [hunk 1]
```diff
@@ -32,10 +37,30 @@ export default function Block({ block }: IProps) {
   const downloadUrl = `data:text/plain;base64,${objectBase64}`;
   const downloadFilename = `3dpass-${block.block.header.number.toString()}.obj`;
 
+  const [isFinalized, setIsFinalized] = useState(false);
+
+  useEffect(() => {
+    setIsFinalized(block.block.header.number.toNumber() <= bestNumberFinalized);
```
Comment (fedosov) [inline app/components/block/Block.client.tsx:43]: Let's extract the block number into a const. It is used multiple times.

File: app/components/common/ExplorerForward.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const BASE_URL = "https://explorer.3dpass.org";
```
Comment (fedosov) [inline app/components/common/ExplorerForward.tsx:1]: Do you think we should move this to our config where an API URL already exists?

File: pnpm-lock.yaml
Comment (fedosov) [inline pnpm-lock.yaml:1]: No need to prettify this file. It is generated automatically. Please revert to the previous state.

**Review by fedosov — CHANGES_REQUESTED** [review]

File: app/components/common/ExplorerForward.tsx (hunk 2 above)
Comment (WlinkNET) [reply app/components/common/ExplorerForward.tsx:1]: We can but I will leave it here bcs will build all stuff if forward explorer on one place.

File: app/components/block/Block.client.tsx (hunk 1 above)
Comment (WlinkNET) [reply app/components/block/Block.client.tsx:43]: Sure I saw that and also want to ask you that,

File: pnpm-lock.yaml
Comment (WlinkNET) [reply pnpm-lock.yaml:1]: Its not prettify it was corrupted when I rebase it

Comment (fedosov) [reply pnpm-lock.yaml:1]: Okay, please take the current file from main branch so there is not diff

Comment (WlinkNET) [reply pnpm-lock.yaml:1]: ok

File: app/components/block/Block.client.tsx (hunk 1 above)
Comment (WlinkNET) [reply app/components/block/Block.client.tsx]: I added const blockNumber = block.block.header.number.toString();

**Pull request merged**